mender.go

```diff
@@ -357,15 +357,19 @@ func (m *mender) FetchUpdate(url string) (io.ReadCloser, int64, error) {
 // that occurred. If no update is available *UpdateResponse is nil, otherwise it
 // contains update information.
 func (m *mender) CheckUpdate() (*client.UpdateResponse, menderError) {
-	currentArtifactName := m.GetCurrentArtifactName()
-	//TODO: if currentArtifactName == "" {
-	// 	return errors.New("")
-	// }
+	currentArtifactName, err := m.GetCurrentArtifactName()
+	if err != nil {
+		log.Error("could not get the current artifact name")
```

Comment:
No, I think this is ok. For delta updates we may have to change this, but we'll cross that bridge later.